### PR TITLE
fix: Timeline UI Changes

### PR DESCRIPTION
**Pull request by VAIBHAV-25**

### CODE REVIEW

File: src/components/Post/Tabs/PostTimeline/TimelineContainer.tsx [hunk 1]
```diff
@@ -25,87 +41,135 @@ interface ITimelineContainerProps {
...
+		displayIconActive = <DiscussionIcon className="-ml-[6px] mr-3 mt-2"/>;
+		displayIconUnactive = <DiscussionIconGrey className="-ml-[6px] mr-3 mt-2" />;
+	}
 	const url = getBlockLink(network);
 
-	const TimelineItems = (isMobile:boolean) => {
-
-		return (
-			<section className={`flex-1 flex ${isMobile? 'flex-col items-start gap-y-20 py-20': 'items-center'}`}>
+	const Timeline = () => {
```
Comment (KanishkaRajputd) [inline src/components/Post/Tabs/PostTimeline/TimelineContainer.tsx:101]: did we create a ui for which in responsive and web, mobile friendly?

Comment (VAIBHAV-25) [reply src/components/Post/Tabs/PostTimeline/TimelineContainer.tsx:101]: yes

File: src/components/Post/Tabs/PostTimeline/TimelineContainer.tsx [hunk 2]
```diff
@@ -25,87 +41,137 @@ interface ITimelineContainerProps {
...
-			<div className='min-h-[300px] bg-pink_primary w-[2px] relative'>
+		<section className='flex my-12 timeline-container'>
+			<div className={`${isCollapsed ? 'min-h-[40px]' : `min-h-${minHeight}`} -mb-[2px] mt-[16px] w-[2px] relative -ml-2`} style={{ backgroundColor: activeColor }}>
 				<Link href={`/${getSinglePostLinkFromProposalType(getFirestoreProposalType(type as any) as any)}/${type === 'Tip'? timeline.hash: timeline.index}`}>
-					<span className='bg-pink_primary rounded-2xl font-medium text-base text-white whitespace-nowrap min-w-[100px] px-5 h-[33px] flex items-center justify-center absolute -left-5 -top-5 shadow-lg shadow-grey_secondary border-.15 border-solid border-pink_primary transition-all ease-in-out  hover:bg-white hover:text-pink_primary'>
-						{getStatus(String(type))}
-					</span>
+					<p className='flex flex-row gap-1 w-[250px] -mt-[40px] font-normal text-base leading-6 whitespace-nowrap h-[33px] -left-[5px] -top-7' style={{ color: activeColor, fontWeight: '500', marginLeft: '-4px' }}>
+						{PostType===timeline.type ? displayIconUnactive  : displayIconActive}
+						<span className='mt-2 font-medium text-base'>{getStatus(String(type))}</span>
```
Comment (KanishkaRajputd) [inline src/components/Post/Tabs/PostTimeline/TimelineContainer.tsx:156]: remove String

File: styles/globals.css [hunk 3]
```diff
@@ -374,4 +374,82 @@ color: var(--lightBlue) !important;
...
+.timeline-dropdown {
+  margin-left: 664px;;
+}
+
+.info-container{
+  min-width: 140px;
+}
+
+.round-icon{
+  margin-left: 60px;
```
Comment (KanishkaRajputd) [inline styles/globals.css:400]: please remove these css form here

**Pull request merged**